src/main/java/com/rcv/GuiApplication.java

```diff
@@ -28,10 +32,18 @@
 public class GuiApplication extends Application {
 
   @Override
-  public void start(Stage window) throws Exception {
-    Parent root = FXMLLoader.load(getClass().getResource("/com/rcv/GuiConfigLayout.fxml"));
-    window.setTitle("RCVRC Tabulator");
-    window.setScene(new Scene(root));
+  public void start(Stage window) {
+    String resourcePath = "/com/rcv/GuiConfigLayout.fxml";
+    try {
+      Parent root = FXMLLoader.load(getClass().getResource(resourcePath));
+      window.setTitle("RCVRC Tabulator");
+      window.setScene(new Scene(root));
+    } catch (IOException exception) {
+      StringWriter sw = new StringWriter();
+      PrintWriter pw = new PrintWriter(sw);
```

Comment:
What's up with these fancy new `StringWriter` and `PrintWriter` libraries, and why are we only using them in the two FXML cases?

Reply:
See my comment in #141.  We use them here because you requested them, and these particular errors should only occur during development, in which case a stack trace is reasonable to log.  In other cases where the exception is expected and handled, or never happens, there is no reason to print a stack trace.

Reply:
Yeah, got it, I was just wondering why it required bringing those new libraries in, rather than just using the standard exception stacktrace functionality. No big deal though, I'm sure you tested it and the normal method wasn't working for some reason.